circleguard/update_check.py

```diff
@@ -0,0 +1,25 @@
+import requests
```

Comment:
any reason this method can't go in utils?

Reply:
dependency loop (settings needs utils, utils (would) need settings)